docs/adopters.md

```diff
@@ -0,0 +1,77 @@
+---
+title: TiKV Adopters
+category: adopters
+---
+
+# Tikv Adopters
+
+This is a list of TiDB adopters in various industries.
+
+|Company| Industry| Success Story| 
+| --- | --- | --- |
+|[Mobike](https://en.wikipedia.org/wiki/Mobike)|Ridesharing|[English](https://www.pingcap.com/blog/Use-Case-TiDB-in-Mobike/); [Chinese](https://www.pingcap.com/cases-cn/user-case-mobike/)|
```

Comment:
do we need to sort the company by name?

Reply:
Let's just keep it as it is.  Kevin reordered them in terms of brand recognition.